tests/modules/i18n/input-method/fcitx5-configuration.nix

```diff
@@ -6,6 +6,7 @@
   i18n.inputMethod = {
     enabled = "fcitx5";
     fcitx5.addons = with pkgs; [ fcitx5-chinese-addons ];
+    fcitx5.waylandFrontend = true;
```

Comment:
I'm not sure how to test that environment variables are not set when by fcitx5 when waylandFrontend is set to true, so I just added that option to the existed test. Please guide me, if there is a good way to approach this test.

Reply:
The most straight forward way would probably be to use `assertFileNotRegex`. An example: https://github.com/nix-community/home-manager/blob/5d151429e1e79107acf6d06dcc5ace4e642ec239/tests/modules/programs/gradle/empty-settings.nix#L12-L15